From 32a7f8aa5fea8eb591c0ad480a7aee59b9a2e526 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Tue, 11 Jul 2017 11:19:49 +0800
Subject: [PATCH 2118/5242] MLK-15949-2 ARM64: dts: fsl-imx8qxp: correct USDHC
 per clock rate

commit  7efe4adcd3b48e80331f9e6451947c11aa3f8967 from
https://source.codeaurora.org/external/imx/linux-imx.git

Set USDHC2/3 per clock's parent clock to 200MHz.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 421b180..a4ea4cf 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1515,7 +1515,8 @@
 			<&clk IMX8QXP_SDHC1_CLK>,
 			<&clk IMX8QXP_CLK_DUMMY>;
 		clock-names = "ipg", "per", "ahb";
-		assigned-clock-rates = <400000000>, <200000000>, <0>;
+		assigned-clocks = <&clk IMX8QXP_SDHC1_DIV>;
+		assigned-clock-rates = <200000000>;
 		power-domains = <&pd_conn_sdch1>;
 		fsl,tuning-start-tap = <20>;
 		fsl,tuning-step= <2>;
@@ -1531,7 +1532,8 @@
 			<&clk IMX8QXP_SDHC2_CLK>,
 			<&clk IMX8QXP_CLK_DUMMY>;
 		clock-names = "ipg", "per", "ahb";
-		assigned-clock-rates = <400000000>, <200000000>, <0>;
+		assigned-clocks = <&clk IMX8QXP_SDHC2_DIV>;
+		assigned-clock-rates = <200000000>;
 		power-domains = <&pd_conn_sdch2>;
 		status = "disabled";
 	};
-- 
1.7.9.5

